### PR TITLE
Split REST Client implementation class

### PR DESCRIPTION
**Pull request by Beacze**
Preparation for search package

### CODE REVIEW

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/KitodoRestClient.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+     * Close REST Client.
+     *
+     * @throws IOException
+     *             add description
+     */
+    public void closeClient() throws IOException {
+        restClient.close();
+    }
+
+    public String getIndex() {
```
Comment (henning-gerhardt) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/KitodoRestClient.java]: public method without JavaDoc

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/KitodoRestClient.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+     * Setter for index.
+     *
+     * @param index
+     *            - equal to the name of database, default kitodo
+     */
+    public void setIndex(String index) {
+        this.index = index;
+    }
+
+    public String getType() {
```
Comment (henning-gerhardt) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/KitodoRestClient.java]: public method without JavaDoc

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/KitodoRestClient.java (hunk 1 above)
Comment (Beacze) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/KitodoRestClient.java]: added

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/KitodoRestClient.java (hunk 2 above)
Comment (Beacze) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/KitodoRestClient.java]: added

**Pull request merged**